Added required shared docs and updated old docs (#213)
exercism/purescript: exercises/shared/.docs/tests.md

> # Tests
> 
> The PureScript exercise test suites may be run from the exercise directory
> using the spago tool:
> 
>     cd <purescript-exercism>/exercises/<exercise-type>/<exercise name>
>     spago test
> 
> where <purescript-exercism> is the root directory where you checked out this
> repository.
> 
> For example:
> 
>     cd ~/purescript/purescript-exercism/exercises/practice/accumulate
>     spago test

Reviewer: The contents of this file will only be present within the exercise directory that is downloaded via the CLI. As such there is no repository context and the file structure is different. You can probably simplify this file to:

```suggestion
# Tests

The test suite can be run using the spago tool:

    spago test
```

Author: I'll update.

Author: Makes sense to make `docs/TESTS.md` and this file the same, agreed? Should I go with this simplified version in both or the longer version from `docs/TESTS.md` in both?

Reviewer: @icyrockcom No they can be different. The `docs/TESTS.md` file is displayed on the website and has no exercise as its context. It can be a bit more verbose. This file (`exercises/shared/.docs/tests.md`) should be concise and to the point and works from the context of an exercise.
